src/test/resources/features/SubmitReview.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Submit Review
+  In order to use the App
+  As a buyer
+  I want to submit a review to a seller
+
+  Background: I'm logged in
+    Given There is no registered user with username "user0"
+    And I'm not logged in
+    When I register a new user with username "user0", email "user@sample.app" and password "password"
+    Then The response code is 201
+    And It has been created a user with username "user0" and email "user@sample.app", the password is not returned
+    And I can login with username "user0" and password "password"
```

Comment:
Background is like a `Given` that will be executed before each test. It's not necessary to do checks with `When` and `Then`